packages/apps/kadena-docs/src/components/Layout/components/Aside/ListItem.tsx

```diff
@@ -0,0 +1,71 @@
+import { AsideLink, AsideList } from './';
+
+import { ISubHeaderElement } from '@/types/Layout';
+import { createSlug } from '@/utils';
+import { analyticsEvent, EVENT_NAMES } from '@/utils/analytics';
+import { useRouter } from 'next/router';
+import React, { FC, MouseEvent } from 'react';
+
+interface IProps {
+  // eslint-disable-next-line @rushstack/no-new-null
+  scrollArea: HTMLDivElement | null;
+  item: ISubHeaderElement;
+  setActiveItem: React.Dispatch<React.SetStateAction<string>>;
+  activeItem?: string;
+}
+
+export const ListItem: FC<IProps> = ({
+  item,
+  setActiveItem,
+  activeItem,
+  scrollArea,
+}) => {
+  const router = useRouter();
+
+  if (item.title === undefined) return null;
+  const slug = `#${createSlug(item.title)}`;
```

Comment:
should we also check the title for ''

the createSlug also only checks undefined